internal/googlecloud/manufacturer_linux.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Build constraints missing in this and the windows file.

Reply:
No build constraints are needed because of the file names. From `go help buildconstraint`:

>If a file's name, after stripping the extension and a possible _test suffix,
>matches any of the following patterns:
>	*_GOOS
>	*_GOARCH
>	*_GOOS_GOARCH
>(example: source_windows_amd64.go) where GOOS and GOARCH represent
>any known operating system and architecture values respectively, then
>the file is considered to have an implicit build constraint requiring
>those terms (in addition to any explicit constraints in the file).

Reply:
Hmm, that sounds right.
But I thought I saw build failures due to duplicate functions.

Reply:
This build failed: https://github.com/grpc/grpc-go/runs/5577974622?check_suite_focus=true

```
# google.golang.org/grpc/internal/googlecloud
Error: internal/googlecloud/manufacturer_linux.go:27:6: manufacturer redeclared in this block
	previous declaration at internal/googlecloud/manufacturer.go:23:30
Error: internal/googlecloud/manufacturer_linux.go:28:9: undefined: os.ReadFile
# google.golang.org/grpc/internal/googlecloud
Error: vet: internal/googlecloud/manufacturer_linux.go:27:6: manufacturer redeclared in this block
```

Reply:
See below, this should be fixed.